frontend/src/components/editor/DocumentView.tsx

```diff
@@ -32,10 +31,21 @@ function DocumentView() {
 									display: "flex",
 									height: "100%",
 									overflow: "hidden",
+									position: "relative",
 								}}
 							>
-								<div id="left-block" style={{ width }}>
-									<Editor />
+								<div
+									id="left-block"
+									style={{
+										width,
+										position: "relative",
+										height: "100%",
+									}}
+								>
+									<div style={{ height: `calc(100% - ${BOTTOM_BAR_HEIGHT}px)` }}>
+										<Editor />
+									</div>
+									<EditorBottomBar width={width} />
```

Comment:
How about moving it to the Editor component?
It seems like there’s some redundancy